sqlparser/parse_test.go

```diff
@@ -215,6 +215,11 @@ var (
 		input: `select * from mytable where "AGE" = 1 and "TEST" = 'test'`,
 		// postgres allow to use double quote string for columns
 		dialect: postgresql.NewPostgreSQLDialect(),
+	}, {
+		input:  `insert into some_table(id, data) VALUES (10918, "test")`,
```

Comment:
lets add comment, that it is temporary valid for MySQL default settings, but invalid for PostgreSQL & MySQL with ANSI mode, and probably will be changed. It will help in the future to understand why this case will fail after future changes)

Reply:
Sure, also added TODO in `invalidSQL` tests that such query should be tests on failure